src/makeRobot.js

```diff
@@ -38,7 +38,67 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}`
+      + `, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+
+      return this;
+    },
+
+    goForward(steps = 1) {
+      if (steps <= 0) {
+        return this;
+      }
+      this.coords.y += steps;
+
+      return this;
```

Comment:
let's make code simpler, fix in all places
```suggestion
      if (steps > 0) {
        this.coords.y += steps;
      }
  
      return this;
```

Reply:
Fixed